samples/getting-started/ReactiveDemo/NugetDetailsView.xaml.cs

```diff
@@ -4,11 +4,14 @@
 
 namespace ReactiveDemo
 {
-    // The class derives off ReactiveUserControl which contains the ViewModel property.
+    /// Nongeneric class that allows us to edit NugetDetailsView in Design view
+    public class NugetDetailsViewBase : ReactiveUserControl<NugetDetailsViewModel> { }
+
+    // Second level derived class off ReactiveUserControl which contains the ViewModel property.
     // In our MainWindow when we register the ListBox with the collection of 
     // NugetDetailsViewModels if no ItemTemplate has been declared it will search for 
     // a class derived off IViewFor<NugetDetailsViewModel> and show that for the item.
-    public partial class NugetDetailsView : ReactiveUserControl<NugetDetailsViewModel>
```

Comment:
Where's this NugetDetailsViewBase come from. Seems redundant and unneccessary and the old approach was better.

Reply:
Fact is that NugetDetailsView.xaml cannot be edited in view designer. There is an issue in VS that does not support editing controls directly inherited from a generic class, so I proposed using a non generic class in between.
`public class NugetDetailsViewBase : ReactiveUserControl<NugetDetailsViewModel> { }`

`public partial class NugetDetailsView : NugetDetailsViewBase`
`{`
`    public NugetDetailsView()`
`    {...`
Other than that, I agree that the class is redundant from a functional point of view. However it is a matter of taste which approach is better - for a beginner, a MVVM framework with a sample project in which "a View cannot be viewed" may not look very promising.

Nevertheless, I have removed the intermediate class as requested.